### ISSUE TITLE
"Click to add cell" button breaks command mode when it's focused

### ISSUE THREAD

**Issue by weibeld** [jupyter/notebook#7118]
## Description

"Click to add cell" button breaks command mode when navigating past the last cell with `J`.

When navigating past the last cell, the "Click to add cell" button appears and gets focused. After that, it's not possible anymore to move the focus to another cell with e.g. `K` or any other key combination.

## Reproduce

1. Open new Jupyter notebook
2. Create new cell with `B`
3. Navigate past last cell with `J`, `J`

The "Click to add cell" button appears and no command mode key combinations can be used anymore.

## Expected behavior

Should be able to navigate back to previous cells with `K`.

## Context

Versions:

- Jupyter:
    ```
    Selected Jupyter core packages...
    IPython          : 8.15.0
    ipykernel        : 6.25.1
    ipywidgets       : not installed
    jupyter_client   : 8.3.0
    jupyter_core     : 5.3.1
    jupyter_server   : 2.7.0
    jupyterlab       : 4.0.5
    nbclient         : 0.8.0
    nbconvert        : 7.7.3
    nbformat         : 5.9.2
    notebook         : 7.0.2
    qtconsole        : 5.4.4
    traitlets        : 5.9.0
    ``` 
- Browser: Google Chrome 117.0.5938.149
- OS: macOS 14.0

Related:

- https://github.com/jupyter/notebook/issues/6852
- https://github.com/jupyterlab/jupyterlab/issues/15224
- https://stackoverflow.com/questions/77245489/jupyter-notebook-click-to-add-a-cell-breaks-commandmode

**Comment by krassowski** [comment]
@weibel this was fixed in [v7.0.6](https://github.com/jupyter/notebook/releases/tag/v7.0.6) released over a week ago, but you are using 7.0.2. Please upgrade your installation.

**Comment by weibeld** [comment]
Indeed it's fixed in Jupyter Notebook v7.0.6, thanks
